### PR TITLE
feat(resource): enable support for tags in cognito userpools & pipes

### PR DESCRIPTION
**Pull request by MSchouwPablo**

### CODE REVIEW

File: resources/cognito-userpools.go
Comment (ekristen) [inline resources/cognito-userpools.go:1]: Can you adjust the imports to the following styling format ...

```
import (
	"context"
	"fmt"

	"github.com/gotidy/ptr"
	"github.com/sirupsen/logrus"

	"github.com/aws/aws-sdk-go/aws"
	"github.com/aws/aws-sdk-go/service/cognitoidentityprovider"
	"github.com/aws/aws-sdk-go/service/sts"
	"github.com/aws/aws-sdk-go/service/sts/stsiface"

	"github.com/ekristen/libnuke/pkg/registry"
	"github.com/ekristen/libnuke/pkg/resource"
	"github.com/ekristen/libnuke/pkg/types"

	"github.com/ekristen/aws-nuke/v3/pkg/nuke"
)
```

File: resources/pipes-pipe.go
Comment (ekristen) [inline resources/pipes-pipe.go:1]: Can you please adjust the inputs to the following styling format.

```
import (
	"context"
	"time"

	"github.com/sirupsen/logrus"

	"github.com/aws/aws-sdk-go/service/pipes"

	"github.com/ekristen/libnuke/pkg/registry"
	"github.com/ekristen/libnuke/pkg/resource"
	"github.com/ekristen/libnuke/pkg/types"

	"github.com/ekristen/aws-nuke/v3/pkg/nuke"
)
```

**Review by ekristen — CHANGES_REQUESTED** [review]
@MSchouwPablo can you run the following. 

```
git reset HEAD~6
git add resources/cognito*
git commit -m 'feat(cognito): add tags to properties'
git add resources/pipes-pipe.go
git commit -m 'feat(pipes-pipe): add tags to properties'
```

OR you can do the same HEAD~6 reset and commit 1 file at a time with `feat(cognito-identity-provider): add support for tags to properties` for each resource type. I'm fine with either approach.

This will also make commit-lint pass as all commits need to be [semantic commit](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716) format.

If you rather, I can do the commit adjustment and become co-author. Up to you.

**Comment by MSchouwPablo** [comment]
@ekristen added your suggestions, but the old commits are still not in the right format. I'm fine with you fixing it, i'll try to do it right from the start next time :D

**Review by ekristen — APPROVED** [review]

**Comment by ekristen** [comment]
> @ekristen added your suggestions, but the old commits are still not in the right format. I'm fine with you fixing it, i'll try to do it right from the start next time :D

git rebasing can be tricky, it's an art form. All is fixed up. 

I can always do a squash merge, but only like to do that when commits are related to a resource maybe 2 if in the same service.

**Pull request merged**